### ISSUE TITLE
Not working 

### ISSUE THREAD

**Issue by senolfndk** [YajanaRao/Serenity#152]
Ferd is not loading

**Comment by senolfndk** [comment]
New version does not open

**Comment by YajanaRao** [comment]
@senolfndk Checking now

**Comment by senolfndk** [comment]
> @senolfndk Checking now. You installed from github releases right ?

I installed it but it still won't open 😒

**Comment by senolfndk** [comment]
It's collapsing

**Comment by YajanaRao** [comment]
> It's collapsing

What is collapsing ?

**Comment by owencz1998** [comment]
I think he means it opens then closes instantly as mine Is literally not opening lol @YajanaRao

**Comment by YajanaRao** [comment]
There is some issue with production build which is happening via github actions. Builds on local are working fine

**Comment by owencz1998** [comment]
Can you send a working apk maybe or is this just whiles your working on the app dude @YajanaRao

**Comment by YajanaRao** [comment]
@owencz1998 checkout the latest release v11.1. This should work as expected. I had to manually create a release for this.

**Comment by YajanaRao** [comment]
@senolfndk The latest release should work for you as well.

**Comment by senolfndk** [comment]
It's crashing again

**Comment by senolfndk** [comment]
It's crashing again

**Comment by owenconnorz** [comment]
Yeah still broken @YajanaRao I even deleted the app and reinstalled it to see if that would fix it up nah still will not open

**Comment by YajanaRao** [comment]
Sad. Let me check

**Comment by YajanaRao** [comment]
@senolfndk @owencz1998 I created a release to fix this issue. Can you please check and reopen the issue if it is still crashing.

**Comment by owenconnorz** [comment]
@YajanaRao still broken dude

**Comment by senolfndk** [comment]
Same problem ![Screenshot_20240819-205554_Lawnchair.jpg](https://github.com/user-attachments/assets/1444b414-6fd7-40b3-b6fc-93fb83fe62d1)

**Comment by YajanaRao** [comment]
What device are you using ? Give me some more details on what's happening

**Comment by owenconnorz** [comment]
@YajanaRao Samsung s24 ultra one ui 6.1 android 14

**Comment by owenconnorz** [comment]
Just opens then closes

**Comment by senolfndk** [comment]
> What device are you using ? Give me some more details on what's happening 

Samsung a21s 
Android 12

**Comment by owenconnorz** [comment]
https://github.com/user-attachments/assets/c6d2e2ed-2d0f-4275-9e2e-f0fb01187746

**Comment by YajanaRao** [comment]
Did you download latest release from GitHub ?

It is working on my device 
![Screenshot_2024-08-19-23-57-01-421_com.serenity.jpg](https://github.com/user-attachments/assets/effcbf1c-c5ac-4201-a1e5-ab27135f9770)

**Comment by senolfndk** [comment]
> Did you download latest release from GitHub ?
> 
> It is working on my device 
> ![Screenshot_2024-08-19-23-57-01-421_com.serenity.jpg](https://github.com/user-attachments/assets/effcbf1c-c5ac-4201-a1e5-ab27135f9770)
> 
> 

Yes

**Comment by senolfndk** [comment]
https://github.com/user-attachments/assets/950d2a4b-7d5e-425d-a619-7f83c0c48583

**Comment by owenconnorz** [comment]
Yeah it's the latest release dude @YajanaRao

**Comment by senolfndk** [comment]
https://github.com/user-attachments/assets/bb08c5d1-7644-43dc-b27f-398ff0e3fd32
